docs/runbooks/mitx_residential_upgrade.md

```diff
@@ -23,3 +23,57 @@ MITx Residential upgrade typically takes place twice a year following OpenedX's
 - Remove old instances from ELB
 - Verify that site is up and functional by running some manual tests
 - Change DNS back to point to ELB
+
+## Creating AMIs
+
+New AMIs are built from the latest Git release branch in our [mitodl/edx-platform](https://github.com/mitodl/edx-platform) repo. This release branch is the edX code plus some patches of our own, specific to our organization; created by rebasing on top of their branch for a particular release.
+
+edX release branches as they appear in edX's upstream repository are named starting with `open-release`; so, for example, the Juniper Release Candidate 3 branch is `upstream/open-release/juniper.rc3` (where my edX upstream repo is named `upstream`).
+
+Our release branches are named starting with `mitx/`, so our edX Juniper release branch, from which our AMIs are built, is `origin/mitx/juniper` (where my MITx origin repo is named `origin`).
+
+Though we build our AMIs from the `mitx/<release codename>` branch, we also keep copies of edX's `open-release` branches for convenience in issuing pull requests against the corresponding branches in `upstream`.
+
+### Updating Git branches
+
+#### The "mitx" release branch
+
+First, a branch is created in our repository for the `mitx/<code name>` release branch for our AMIs:
+
+```
+git checkout -b mitx/juniper upstream/open-release/juniper.rc1
+git push -u origin mitx/juniper
+```
+
+This is usually branched off of edX's first release candidate branch for the relevant release codename.
```

Comment:
We typically wait for the `.master` branch to be created for a given release. Juniper is a special case because it took so long to get to the release.

Reply:
I've changed this in commit f4fa661f